vg/vgsvg/vgsvg.go

```diff
@@ -73,11 +77,13 @@ func New(w, h vg.Length) *Canvas {
 func NewWith(opts ...option) *Canvas {
 	buf := new(bytes.Buffer)
 	c := &Canvas{
-		svg:   svgo.New(buf),
-		w:     DefaultWidth,
-		h:     DefaultHeight,
-		buf:   buf,
-		stack: []context{{}},
+		svg:    svgo.New(buf),
+		w:      DefaultWidth,
+		h:      DefaultHeight,
+		buf:    buf,
+		stack:  []context{{}},
```

Comment:
Please make this `make([]context, 1)` if you want an explicit unit-length slice.

Reply:
That's not my code. It was there before. The line was modified because of the alignment change. 

I'm not sure what a context is. I suspect it is a shorthand notation to instantiate a slice of slice structure, or something like that.

Reply:
You were right. I fixed it.